multinode-demo/remote_env.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+usage() {
+  echo -e "\\tUsage: source this-script <IP Address array> <username> [path to ssh keys]\\n"
```

Comment:
How about this form?  https://github.com/solana-labs/solana/blob/master/multinode-demo/setup.sh#L7-L28

I also usually like my `usage` to exit for me, since 99.9% of the time the next command after `usage` is `exit`